src/react/utils/hooks.ts

```diff
@@ -84,4 +86,14 @@ export const useClipboard = () => {
     copy: copyToClipboard,
     copyStatus: copyStatus,
   };
-};
\ No newline at end of file
+};
+
+export const useAccounts = () => {
+  const accounts = useSelector(
+    (state: AppState) => state.configuration.latest.users,
+  );
+
+  return accounts.filter(
```

Comment:
Maybe we can filter by account Id because the userName can be edited one day ?

Reply:
Hi @AsmaaELM 
Just in case, I filter out the account by either the userName or the id.